api/src/test/java/org/apache/iceberg/transforms/TestTruncate.java

```diff
@@ -86,4 +87,12 @@ public void testTruncateByteBuffer() throws Exception {
         ByteBuffer.wrap("abc".getBytes("UTF-8")),
         trunc.apply(ByteBuffer.wrap("abc".getBytes("UTF-8"))));
   }
+
+  @Test
+  public void testVerifiedIllegalWidth() {
+    AssertHelpers.assertThrows("Should fail if width is less than or equal to zero",
+        IllegalArgumentException.class,
+        "The width of truncate must larger than zero",
```

Comment:
`must larger` -> `must be larger`, here and in other places with similar message